src/os2datascanner/projects/admin/adminapp/templates/miniscan.html

```diff
@@ -56,6 +68,16 @@ <h2>{% trans "File to scan" %}</h2>
               {% trans "Please, select another (smaller) file to scan." %}
             </div>
           {% endwith %}
+
+          <h2>{% trans "Enter your text below" %}</h2>
+          <textarea name="text"
+                 id="upload-text"
+                 type="text"
+                 class="upload"
+                 spellcheck="true"
+                 maxlength="2000"> <!-- Hardcoded maxlength to 2000, maybe add {{ textLimit }} or something similar -->
```

Comment:
I didn't actually know this, but `<textarea>` is one of those odd tags like `<pre>` that doesn't respect comments:

![image](https://github.com/magenta-aps/osdatascanner/assets/45562185/d10cf412-80d5-41cf-9f88-cae5e2c44573)

I suggest putting the comment immediately before the `<textarea>` tag.

Reply:
Odd. I should have tested before doing git commit. thanks for pointing this out.